scripts/run_single_window_test.py

```
@@ -0,0 +1,223 @@
+            if up_bid is not None and up_ask is not None:
+                mid = (up_bid + up_ask) / 2.0
+
+            if mid > 0.50:
+                max_up_drift = max(max_up_drift, mid - 0.50)
+            else:
+                max_down_drift = max(max_down_drift, 0.50 - mid)
+
+            # Check Fills directly on Polymarket CLOB
+            if not filled_up and up_id:
+                try:
+                    ord_up = clob_client.get_order(up_id)
```

Ruff (0.16.3)

[error] Undefined name `clob_client`

(F821)